pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/RealtimeProvisioningHelperCommand.java

```diff
@@ -71,9 +72,15 @@
   @Option(name = "-numHours", metaVar = "<String>", usage = "number of hours to consume as comma separated values (default 2,3,4,5,6,7,8,9,10,11,12)")
   private String _numHours = "2,3,4,5,6,7,8,9,10,11,12";
 
-  @Option(name = "-sampleCompletedSegmentDir", required = true, metaVar = "<String>", usage = "Consume from the topic for n hours and provide the path of the segment dir after it completes")
+  @Option(name = "-sampleCompletedSegmentDir", required = false, metaVar = "<String>", usage = "Consume from the topic for n hours and provide the path of the segment dir after it completes")
   private String _sampleCompletedSegmentDir;
 
+  @Option(name = "-dataCharacteristicsFile", required = false, metaVar = "<String>", usage = "File containing characteristics by which a segment will be generated")
+  private String _dataCharacteristicsFile;
+
+  @Option(name = "-schemaFile", required = false, metaVar = "<String>")
```

Comment:
We should not provide schema file. See my comment further below. The data characteristics file is essentially the schema annotated with additional info. So it alone should be enough

Reply:
Isnt it good to have the option of providing  a sample segment OR data characteristics?

Reply:
@mcvsubbu , sorry I don't think I am following this comment? We are providing data characteristics right